src/views/mixins/footer.pug

```diff
@@ -0,0 +1,17 @@
+mixin  footer()
+    .Footer
+        .Brand 
+            img(src="favicon.ico", alt="JetSavr Logo")
+            a(href="", rel="noopener noreferrer") About Us 
+            a(href="", rel="noopener noreferrer") Contact Us 
+
+        .Nav-Links 
+            a(href="/") Home 
+            a(href="" , rel="noopener noreferrer") Services 
+            a(href="", rel="noopener noreferrer") Blog/News
+        
+        .Github  
+            h4 Contributors
+             a(href="https://github.com/william-man", target="_blank", rel="noopener noreferrer")  William
+            a(href="https://github.com/halimahexe", target='__blank' rel="noopener noreferrer") Halimah
```

Comment:
Please can you correct `target='__blank'` to `target='_blank'`?

Reply:
fixed typo